data/lib/service/match/match_service.dart

```diff
@@ -464,4 +464,18 @@ class MatchService {
       throw AppError.fromError(error, stack);
     }
   }
+
+  //Helper Methods
```

Comment:
let's remove this helper method tag from here, we use this tag for methods that gets data from another service class to one service class.